src/pool/srv_pool.c

```diff
@@ -4542,13 +4542,20 @@ ds_pool_upgrade_if_needed(uuid_t pool_uuid, struct rsvc_hint *po_hint,
 	rdb_tx_end(&tx);
 out_svc:
 	if (upgraded) {
+		if (rc == 0 && need_put_leader &&
+		    DAOS_FAIL_CHECK(DAOS_POOL_UPGRADE_CONT_ABORT)) {
+			D_DEBUG(DB_MD,
+				DF_UUID": fault injected: DAOS_POOL_UPGRADE_CONT_ABORT\n",
+				DP_UUID(svc->ps_uuid));
```

Comment:
DAOS_FAIL_CHECK will log this anyway, do you need an extra debug point here?

Reply:
Good point, let me remove it.